lib/settings/persistence/local_storage_settings_persistence.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+import 'settings_persistence.dart';
+
+class LocalStorageSettingsPersistence extends SettingsPersistence {
+  final Future<SharedPreferences> instanceFuture =
+      SharedPreferences.getInstance();
```

Comment:
Could you please describe the decision behind this solution? We have `TrashReserveRepository` which is also using SharedPreferences and we get an instance of `SharedPreferences` in initialize method. Here we each time calling having an async function to get an instance of SharedPreferences

Reply:
Fixed